tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalJBossBundle.java

```diff
@@ -93,6 +93,11 @@ public String getHttpPort()
     }
 
     @Override
+    public void setHttpPort( String port )
```

Comment:
I have told with Simon with it and he will work on jboss things after you merge this tomcat code, and now the server editor wouldn't show the port section if you select jboss server, i add this limit if we have no time to finish jboss stuff for 3.1 ga